opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/authorization/payment/xs2a-authorize-payment.bpmn20.xml

```diff
@@ -88,6 +88,40 @@
     <sequenceFlow id="noOauth2Authorization" sourceRef="sid-3ac85279-91f9-41fc-8644-369b8d2c9ee5" targetRef="startAuthorization">
       <conditionExpression xsi:type="tFormalExpression"/>
     </sequenceFlow>
+    <exclusiveGateway id="isDecoupledScaSelected" name="Is the selected SCA decoupled" default="nonDecoupledScaSelected"/>
+    <sequenceFlow id="nonDecoupledScaSelected" sourceRef="isDecoupledScaSelected" targetRef="embeddedScaChallenge">
+      <conditionExpression xsi:type="tFormalExpression"/>
+    </sequenceFlow>
+    <subProcess id="decoupledScaStatusChecking" flowable:exclusive="true" name="Decoupled Sca status checking subprocess">
+      <serviceTask id="checkIfDecoupledScaFinalized" flowable:exclusive="true" flowable:delegateExpression="${xs2aPisCheckDecoupledAuthorizationStatus}" name="Checks authorization status to be finalized"/>
+      <startEvent id="decoupledSubProcessStart" name="Start of decoupled Sca subprocess"/>
+      <sequenceFlow id="startAuthorizationChecking" sourceRef="decoupledSubProcessStart" targetRef="checkIfDecoupledScaFinalized" name="Start authorization checking"/>
+      <sequenceFlow id="sid-ed27275e-353c-4cf7-92d6-0d017cd2d8f0" sourceRef="checkIfDecoupledScaFinalized" targetRef="isAuthrizationSucceded"/>
+      <endEvent id="subprocessEnd" name="End of decoupled Sca status checking subprocess"/>
+      <sequenceFlow id="finishSubprocess" sourceRef="isAuthrizationSucceded" targetRef="subprocessEnd" name="Successful proceeding to the subprocess end">
+        <conditionExpression xsi:type="tFormalExpression"/>
+      </sequenceFlow>
+      <intermediateCatchEvent id="waitAuthorizationConfirmation" name="Waits until authorization is finished by PSU">
+        <timerEventDefinition>
+          <timeDuration>${flowable.timer.delayDuration}</timeDuration>
+        </timerEventDefinition>
+      </intermediateCatchEvent>
+      <sequenceFlow id="startTimer" sourceRef="isAuthrizationSucceded" targetRef="waitAuthorizationConfirmation" name="Waits for user to authorize in 3rd party using decoupled approach">
```

Comment:
'isAuthrizationSucceded' and typos containing `Authrization` is still present

Reply:
double checked and removed all typos regarding authorization word